api/pom.xml

```diff
@@ -145,6 +145,11 @@
          <groupId>org.apache.lucene</groupId>
          <artifactId>lucene-queries</artifactId>
       </dependency>
+      <dependency>
+				<groupId>mysql</groupId>
+				<artifactId>mysql-connector-java</artifactId>
+				<scope>test</scope>
+			</dependency>
```

Comment:
probably this change here can be another ticket separate from this ticket herein being solved!

Reply:
Alright then....